liquidity/ui/src/components/Deposit/Deposit.tsx

```diff
@@ -268,12 +268,12 @@ export function Deposit() {
         </Alert>
       </Collapse>
 
-      {collateralType && liquidityPosition ? (
+      {collateralType && (liquidityPosition || !params.accountId) ? (
         <Collapse
           in={
             collateralChange.gt(0) &&
             collateralChange
-              .add(liquidityPosition.collateralAmount)
+              .add(liquidityPosition?.collateralAmount || ZEROWEI)
```

Comment:
I really prefer not to use ?. as much as possible and be more explicit

```suggestion
              .add(liquidityPosition ? liquidityPosition.collateralAmount : ZEROWEI)
```

smth like this would be more explicit.